rs-backend/src/git.rs

```diff
@@ -0,0 +1,47 @@
+use std::env;
+use std::path::Path;
+use std::process::Command;
+
+/// Fetches repo path from supplied command line argument, panics otherwise.
+pub fn get_repo_path() -> String {
+    if env::args().count() < 2 {
+        panic!("No argument supplied, needs location of data repo.");
+    }
+    env::args().nth(1).unwrap()
+}
+
+
+pub fn update_repo(repo_path: String) {
+    let mut working_dir = Path::new(&repo_path);
+    if env::set_current_dir(working_dir).is_err() {
+        panic!("Invalid path specified");
+    }
+
+    let fetch = Command::new("git")
```

Comment:
I would issue a `git checkout master` first, just to make sure we are in the correct state